rfcs/any.js-globals.md

```diff
@@ -0,0 +1,25 @@
+# RFC 52: Remove default and negated global support in \*.any.js
+
+## Summary
+
+https://web-platform-tests.org/writing-tests/testharness.html#multi-global-tests has a confusing syntax for specifying globals. The default set of globals is always included implicitly, e.g., `global=window` is equivalent to `global=window,dedicatedworker` and `global=jsshell` is equivalent to `global=window,dedicatedworker,jsshell`.
+
+This RFC will make these changes:
+
+1. Remove `default` (though keep `window,dedicatedworker` as the default if `// META: global=` is not present).
+2. Remove `!` (negation). Instead test developers are expected to either rely on the default or explicitly list the globals they want.
+
+A test supposed to run in dedicated and shared workers would use `dedicatedworker,sharedworker` and not `!window,sharedworker`, `!window,worker,!serviceworker`, etc.
+
+## Details
+
+1. Update all `*.any.js` tests that use `// META: global=` to list all the relevant globals explicitly when possible. (Some necessarily rely on `!default` or `!`.
```

Comment:
```suggestion
1. Update all `*.any.js` tests that use `// META: global=` to list all the relevant globals explicitly when possible. (Some necessarily rely on `!default` or `!`).
```

Reply:
The closing parenthesis should go after the dot in this case.

Reply:
(Fixed.)